specification/ai/OpenAI.Assistants/runs/models.tsp

```diff
@@ -138,6 +138,13 @@ model ThreadRun {
   @added(ServiceApiVersions.v2024_05_01_preview)
   toolChoice: AssistantsApiToolChoiceOption | null;
 
+  /**
+   * Whether to enable parallel function calling during tool use.
+   */
+  @added(ServiceApiVersions.v2024_09_01_preview)
+  @encodedName("application/json", "parallel_tool_calls")
+  parallelToolCalls: boolean = true;
```

Comment:
Isn't this a response model? Why is there a default value?

Reply:
Double-checked OpenAI's spec and they do this, albeit with slight indirection: in the .yaml, `RunObject.parallel_tool_calls` is a `required` `$ref` to the `ParallelToolCalls` component, which in turn is just a named `boolean` with a `true` default.

I think it was done that way out of convenience, though, as `ParallelToolCalls` is reused in both optional and required contexts across both request and response models.

I agree that it makes the most sense to remove the default here in this collapsed form. It almost seems like it should be disallowed by the compiler, even.

Reply:
Default removed by [c9722d5](https://github.com/Azure/azure-rest-api-specs/pull/30909/commits/c9722d55a245b3fcdd21b312c98c578f07e7ca28)